File: pyg-manager-web/src/main/java/top/pyg/manager/controller/LoginController.java
package top.pyg.manager.controller;

import java.util.HashMap;
import java.util.Map;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/login")
public class LoginController {
	/**
	 * 获取当前登录用户信息
	 */
	@RequestMapping("/loadLoginName")
	public Map loadLoginName() {
		String username = SecurityContextHolder.getContext().getAuthentication().getName();
		
		Map map = new HashMap();
		
		map.put("loginName", username);
		
		return map;
	}
	

}

File: pyg-shop-web/src/main/webapp/js/controller/loginController.js
//定义控制器
app.controller("loginController",function($scope, $controller,loginService) {
	//获取登录用户名
	$scope.loadLoginName = function(){
		loginService.loadLoginName().success(
			function(data){
				$scope.loginName = data.loginName;
			}	
		);
	}

})